doc/release/1.13.0-notes.rst

```diff
@@ -200,6 +200,12 @@ All of the following functions in ``np.linalg`` now work when given input
 arrays with a 0 in the last two dimensions: `det``, ``slogdet``, ``pinv``,
 ``eigvals``, ``eigvalsh``, ``eig``, ``eigh``.
 
+``randint`` and ``choice`` can now deal with zero size
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Missing some tildes here. Better title would probably be "work on empty distributions" instead of "can now deal withzero size"